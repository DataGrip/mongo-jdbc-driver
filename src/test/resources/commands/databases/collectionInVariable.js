// before
db.col.insert({a: 1});
var col = db.getCollection('col');
// command
col;
// clear
db.getCollection('col').drop();
