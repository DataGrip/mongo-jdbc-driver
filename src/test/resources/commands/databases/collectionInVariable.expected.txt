|map|
|Document{{_id=<ObjectID>, a=1}}|